app/controllers/game_controller.rb

```diff
@@ -1,35 +1,38 @@
 class GameController < ApplicationController
   skip_before_action :verify_authenticity_token
 
-  # def start
-  def initilize
-    super
-    @lives ||= 9
-    @words ||= %w[programming fun internship ruby rails coding design]
-
-    @word_to_guess ||= @words.sample
-    @hidden_word = @word_to_guess.chars.map { '_' }
+  def start
+    words = %w[programming fun internship ruby rails coding design]
+    game = GameSession.new(word_to_guess: words.sample, guesses: '')
+    game.save
+    render('start')
   end
 
   def word_guess
-    @user_guess = params[:guess] # .chomp.downcase
+    user_guess = params[:guess] # .chomp.downcase
+    game = GameSession.last
+    guesses = game.guesses
+    game.update(guesses: guesses + user_guess)
+    @hidden_word = game.get_hidden_word
 
-    if @word_to_guess.include? @user_guess
-      @hidden_word = @hidden_word.filter_map.with_index do |letter, index|
-        @user_guess == @word_to_guess[index] ? @user_guess : letter
-      end
-    else
-      @lives -= 1
-    end
-    if @hidden_word.join('') == @word_to_guess
-      @message = 'You won the game!'
-      render('result')
+    @lives = game.lives
+    if game.win?
```

Comment:
I reckon two improvements to this. 

1. GameSession determines if it's a win or not. So let's also have it determine if we lose. In the GameSession class:
```ruby
def lose
  lives < 1
end
```
2. I think this would be better as an if/elsif statement, to emphasise that only one option of win/lose/continue is possible. Then you don't need the `return` either.
```ruby
if game.win?
  #
elsif game.lose?
  #
else
 # 
end
```